src/WSDecoder.elm

```diff
@@ -289,6 +290,35 @@ type alias ArtistObj =
     }
 
 
+videoQueryDecoder : Decoder ResultResponse
+videoQueryDecoder =
+    Decode.succeed ResultM
+        |> custom (at [ "result", "videos" ] (list videosDecoder))
+
+
+videosDecoder : Decoder VideosObj
+videosDecoder =
+    Decode.succeed VideosObj
+        |> required "label" string
+        |> required "artist" (list string)
+        |> required "albumid" int
+        |> required "duration" int
+        |> required "videoid" int
+        |> required "thumbnail" string
+        |> required "genre" (list string)
+
+
+type alias VideosObj =
```

Comment:
I think the naming here is misleading, drop the `s` as it's only one video.

Reply:
After doing that, you probably want to cleanup the naming of variables in that regard too

Reply:
I changed the variable name at all required places.